webrecorder/browsertrix
# Frontend localhost icon fix

No issue created, reported here https://github.com/webrecorder/browsertrix-cloud/pull/1534#issuecomment-1987758321

### Changes

Loads `sl-icon` synchronously to get correct base path when running webpack-dev-server.

### Manual testing

Run `yarn --frozen-lockfile` to install latest shoelace. Run `yarn start` and verify icons load.
